README.md

```diff
@@ -458,13 +459,13 @@ policy described at https://cloud.google.com/terms.
 
 Applications developers interact with a C++ library through more than just
 the C++ symbols and headers. They also need to reference the name of the
-library in their build scripts. Depending of the build system they use
+library in their build scripts. Depending on the build system they use
 this may be a CMake target, a Bazel rule, a pkg-config module, or just the
 name of some object in the file system.
 
 As with the C++ API, we try to avoid breaking changes to these interface
 points. Sometimes such changes yield benefits to our customers, in the form of
-easier-to-understand what names go with with services, or more consistency
+easier-to-understand what names go with services, or more consistency
```

Comment:
This clause doesn't really parse.

maybe: "... in the form of bug fixes, easier-to-understand names, or increased consistency across services."

Reply:
Fixed.